utils/prometheus/parser.go

```diff
@@ -0,0 +1,45 @@
+package prometheus
```

Comment:
Can we somehow mark in this file that it was copied from prometheus and the licencing terms this is about?
I would prefer to have a direct link to github with commit id

Reply:
This file contains only functions that I have implemented to have the same effects when parsing the configuration. The original code was not really reusable and far too complex for your needs.